internal/cli/root/atlas/builder.go

```diff
@@ -93,6 +94,13 @@ func Builder(profile *string) *cobra.Command {
 			}
 
 			if shouldCheckCredentials(cmd) {
+				if err := opts.InitFlow(); err != nil {
+					return err
+				}
+
+				if err := opts.RefreshAccessToken(cmd.Context()); err != nil {
+					return err
+				}
 				return validate.Credentials()
```

Comment:
not a strong opinion here (since I've seen both options in the past), but if refreshAccessToken fails do we want to fail the flow? Maybe the existing credentials are still valid. And if the credentials are not valid anymore the flow will be interrupted at by line `return validate.Credentials()`

Reply:
ignore this, checked the RefreshAccessMethod and if the token is valid it doesn't try to refresh it (I thought it will try to refresh it if x% of it's validity passed)
```
	if current.Valid() {
		return nil
	}
```